publiccode.yml

```
@@ -0,0 +1,551 @@
+publiccodeYmlVersion: "0.4"
+
+name: Decidim
+url: "https://github.com/decidim/decidim.git"
+platforms:
+  - web
+developmentStatus: stable
+softwareVersion: "0.29.1"
+releaseDate: "2024-10-02"
+logo: "https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/logo.svg"
+softwareType: "standalone/web"
```

As per description, we should use  `library`:
`standalone/web` - The software represents a web application usable by means of a browser.
`library` - The software contains a library or an SDK to make it easier to third party developers to create new products.

And as far as i can remember, the people using decidim still needs to use command line to generate an app. 

```suggestion
softwareType: "library"
```